src/centralserver/admin-service/application/src/intTest/java/org/niis/xroad/cs/test/glue/IntermediateCasApiStepDefs.java

```diff
@@ -73,6 +74,19 @@ public void ocspResponderIsAddedToIntermediateCAS() throws Exception {
                 .addIntermediateCaOcspResponder(intermediateCasId, url, certificate);
 
         assertEquals(CREATED, response.getStatusCode());
+
+        scenarioContext.putStepData("ocspRespIdOfIntermediateCa", response.getBody().getId());
+    }
+
+    @When("OCSP responder is deleted from intermediate CAS")
```

Comment:
I am little bit confused, is there intermediate CAS and intermediate CA two different thing?